### PR TITLE
chore: Docs updates

### PR DESCRIPTION
**Pull request by pj-sp**
1. Update user agent information 
2. Add section about header auth
3. Update packages

### CODE REVIEW

File: docs/infrastructure/auth/header-auth.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Header Authentication
+
+We support that you put your feeds protected by authentication header, please contact us at support@prisjakt.nu in order to provide us header name and value for your feed.
```
Comment (andrejohansson) [inline docs/infrastructure/auth/header-auth.md:3]: This is actually doable by themselves in BC

image

Comment (pj-sp) [reply docs/infrastructure/auth/header-auth.md:3]: hmm, the same for other types of authentication. Should we update it respectively?

Comment (andrejohansson) [reply docs/infrastructure/auth/header-auth.md:3]: yes, please do so we are clear!

Comment (pj-sp) [reply docs/infrastructure/auth/header-auth.md]: done

**Review by andrejohansson — APPROVED** [review]

**Pull request merged**